Cesium3DTilesSelection/src/TilesetContentManager.cpp

```diff
@@ -663,6 +663,7 @@ TilesetContentManager::TilesetContentManager(
             [pLogger = externals.pLogger,
              asyncSystem = externals.asyncSystem,
              pAssetAccessor = externals.pAssetAccessor,
+             &requestHeaders = _requestHeaders,
```

Comment:
You can get the request headers from the `IAssetRequest` instead of capturing them here.

That's important because capturing them by reference here is a potentially dangerous pattern. We end up sharing the request headers between the thenInWorkerThread lambda and the TilesetContentManager running in the main thread. That's fine as long as neither thread modifies the request headers. If the headers can ever be modified, though, then behavior is undefined, which probably means in this case "an extremely rare and hard to debug crash." I think that can't actually happen currently, but it's a precarious situation as the code base evolves.